lib/ansible/modules/packaging/os/pacman.py

```diff
@@ -433,6 +433,7 @@ def main():
     )
 
     pacman_path = module.get_bin_path('pacman', True)
+    module.run_command_environ_update = dict(LC_ALL='C')
```

Comment:
Could you please add a changelog entry for this change [here](https://github.com/ansible/ansible/tree/devel/changelogs/fragments) so that this change can be backported to 2.9 branch?

You can find the documentation on [How to backport merged PR](https://docs.ansible.com/ansible/latest/community/development_process.html?backporting-merged-prs#backporting-merged-prs)

Let me know once you are done with this change.

Reply:
@Akasurde : I added a changelog fragment.

I can submit the backport PR against 2.9 once this one is one is merged on devel.

Reply:
Backport to 2.9 is in #66045.